config/initializers/teaspoon.rb

```diff
@@ -0,0 +1,7 @@
+unless Rails.env.production?
+  module Teaspoon
+    class SuiteController < ActionController::Base
```

Comment:
Try this:


```ruby
unless Rails.env.production?
  require 'teaspoon/suite_controller'

  module Teaspoon
    class SuiteController
      skip_before_filter :handle_two_factor_authentication
    end
  end
end
```

the reason being if for some reason the Teaspoon gem ever changes the superclass of the SuiteController we don't have to worry about it not matching